src/components/app.jsx

```diff
@@ -1,62 +1,56 @@
 import React from 'react';
-import url from 'url';
 import '../../node_modules/uswds/dist/js/uswds.js';
 import '../../node_modules/@cmsgov/design-system-core/dist/index.css';
+import { Route } from 'react-router-dom'
```

Comment:
i think there are a lot of missing semicolons - i'll add our eslint rules to this project